warehouse/query-core/src/main/java/datawave/query/composite/CompositeTerm.java

```diff
@@ -85,7 +89,17 @@ public void getNodesAndExpressions(List<Class<? extends JexlNode>> nodeClasses,
      * @return
      */
     private String getAppendedExpressions() {
-        return String.join(separator, expressionList);
+        final List<String> newExpressions;
+        if (Iterables.getLast(jexlNodeList) instanceof ASTERNode && jexlNodeList.stream().filter(x -> x instanceof ASTERNode).count() == 1) {
+            // We have an ASTERNode in the last position, but the rest are not ERNodes ( and therefore must be EQNodes )
+            // escape all of the terms
+            newExpressions = expressionList.stream().map(Pattern::quote).collect(Collectors.toList());
+            // set the last ( the regex ) so that it is not escaped.
+            newExpressions.set(expressionList.size() - 1, expressionList.get(expressionList.size() - 1));
+        } else {
+            newExpressions = expressionList;
+        }
+        return String.join(separator, newExpressions);
```

Comment:
Here's a version that goes through the list only once and throws away the quoted list if we encounter an unexpected ASTERNode. Personally, I think what's there is a lot more readable even if it does unnecessarily quote and overwrite the last item in the list.
```suggestion
        List<String> newExpressions = expressionList;
        // We are expecting that the list has either all eq nodes or all eq nodes
        // with one er at the end. If there's an er node at the end, then we need
        // to quote all of the eq nodes so that any regex characters are escaped.
        // If for some reason there's an er node at the end and one or more eq nodes
        // in the middle, then abort and use the original unquoted expression list.
        if (Iterables.getLast(jexlNodeList) instanceof ASTERNode) {
            newExpressions = new ArrayList<>(expressionList.size());
            for (int i = 0; i < jexlNodeList.size() - 1; ++i) {
                if (jexlNodeList.get(i) instanceof ASTERNode) {
                    break;
                } else {
                    newExpressions.add(Pattern.quote(expressionList.get(i)));
                }
            }
            if (newExpressions.size() < expressionList.size() - 1) {
                newExpressions = expressionList;
            } else {
                newExpressions.add(Iterables.getLast(expressionList));
            }
        }
        return String.join(separator, newExpressions);
```

Reply:
Or another way that doesn't overwrite the last item in the list. It's not terrible, but seems kind of like one of those cases of shoehorning something that doesn't fit into the streams API.
```suggestion
        final List<String> newExpressions;
        if (Iterables.getLast(jexlNodeList) instanceof ASTERNode && jexlNodeList.stream().filter(x -> x instanceof ASTERNode).count() == 1) {
            // We have an ASTERNode in the last position, but the rest are not ERNodes ( and therefore must be EQNodes )
            // escape all of the terms except for the last one.
            newExpressions = Stream.concat(
                    expressionList.stream().limit(expressionList.size() - 1).map(Pattern::quote),
                    Stream.of(expressionList.get(expressionList.size() - 1))
                ).collect(Collectors.toList());
        } else {
            newExpressions = expressionList;
        }
        return String.join(separator, newExpressions);
```